snippets/lwc-xml.json

```diff
@@ -0,0 +1,72 @@
+{
+    "Api Version": {
+        "body": [
+            "<apiVersion>$0</apiVersion>"
+        ],
+        "description": "Api Version",
+        "prefix": [
+            "apiVersion"
+        ],
+        "scope": "xml"
+    },
+    "Capability": {
+        "body": [
+            "<capability>${1|lightningCommunity__RelaxedCSP,lightning__ServiceCloudVoiceToolkitApi|}</capability>"
+        ],
+        "description": "Capability",
+        "prefix": [
+            "capability"
+        ],
+        "scope": "xml"
+    },
+    "Is Exposed": {
+        "body": [
+            "<isExposed>${1|true,false|}</isExposed>"
+        ],
+        "description": "Is Exposed",
+        "prefix": [
+            "isExposed"
+        ],
+        "scope": "xml"
+    },
+    "Targets": {
+        "body": [
+            "<targets>",
+            "\t$0",
+            "</targets>"
+        ],
+        "description": "Targets",
+        "prefix": [
+            "targets"
+        ],
+        "scope": "xml"
+    },
+    "Target Config": {
+        "body": [
+            "<targetConfig targets=\"${1|lightningCommunity__Default,lightningCommunity__Page,lightningCommunity__Page_Layout,lightningCommunity__Theme_Layout,lightningSnapin__ChatHeader,lightningSnapin__ChatMessage,lightningSnapin__Minimized,lightningSnapin__PreChat,lightningStatic__Email,lightning__AppPage,lightning__FlowScreen,lightning__HomePage,lightning__Inbox,lightning__RecordAction,lightning__RecordPage,lightning__Tab,lightning__UtilityBar,analytics__Dashboard|}\">",
+            "\t$0",
```

Comment:
Here you can add at least one property, because targetConfig never exist without `propertyType`